connexion/api.py

```diff
@@ -125,6 +125,10 @@ def __init__(self, swagger_yaml_path, base_url=None, arguments=None,
         # API calls.
         self.produces = self.specification.get('produces', list())  # type: List[str]
 
+        # A list of MIME types the APIs can consume. This is global to all APIs but can be overridden on specific
+        # API calls.
```

Comment:
Sorry, what exactly do you mean? The comment relates to the `consumes` param yet it's almost similar to the `produces`. Should I rewrite it somehow or just remove?